package.json

```diff
@@ -149,10 +152,10 @@
     "uuid": "^7.0.1"
   },
   "peerDependencies": {
-    "@types/react": "^16.9.19",
-    "@types/react-dom": "^16.9.5",
-    "@types/react-router": "^5.1.4",
-    "@types/react-router-dom": "^5.1.3",
+    "@types/react": "^17.0.2",
```

Comment:
We try not to increase the peerDependencies to support maximum backward compatibility.